src/Illuminate/Collections/Traits/EnumeratesValues.php

```diff
@@ -195,7 +195,7 @@ public function some($key, $operator = null, $value = null)
     /**
      * Determine if an item exists, using strict comparison.
      *
-     * @param  (callable(TValue): bool)|TValue|string  $key
+     * @param  (callable(TValue): bool)|TValue|array-key  $key
```

Comment:
This change is covered by our tests @canvural?

Reply:
Yes. This two lines are covering this change: https://github.com/laravel/framework/pull/40416/files#diff-61f68115cbe917593bd9c2d4e38338bb6ef9e485739fdff0e989936695c504efR94-R95